internal/sla/sla_test.go

```
@@ -0,0 +1,406 @@
+	rows := fetchApplied(t, db, conv)
+	if len(rows) != 2 {
+		t.Fatalf("expected settled sla kept plus new pending, got %+v", rows)
+	}
+	if rows[0].ID != old.ID || rows[0].Status == "pending" {
+		t.Fatalf("expected old sla closed, got %+v", rows[0])
+	}
+	if rows[1].Status != "pending" {
+		t.Fatalf("expected new pending sla, got %+v", rows[1])
+	}
+	var events, notifs int
+	db.QueryRow(`SELECT COUNT(*) FROM sla_events WHERE applied_sla_id = $1 AND status = 'pending'`, old.ID).Scan(&events)
```

golangci-lint (2.12.2)

[error] Error return value of `(*database/sql.Row).Scan` is not checked

(errcheck)